lib/carrierwave/sanitized_file.rb

```diff
@@ -329,6 +328,22 @@ def sanitize(name)
       return name.mb_chars.to_s
     end
 
+    def existing_content_type
+      if @file.respond_to?(:content_type) && @file.content_type
+        @file.content_type.to_s.chomp
+      end
+    end
+
+    def mime_magic_content_type
+      MimeMagic.by_magic(File.open(path)).try(:type) if path
```

Comment:
This also causes a file handle to be left open. When upgrading from 0.11.0 to 0.11.1, all my upload tests started to fail as I always check all files are closed https://github.com/obduk/cms/blob/master/spec/support/all/bad_tests.rb#L2

I tracked it down to this code, which as also been backported in to 0.11.1 in #1936 

I don't mind fixing it, but don't want to cause collisions with lines that people are already working on?